scriptmodules/emulators/retroarch.sh

```diff
@@ -110,9 +110,14 @@ function update_core_info_retroarch() {
     local dir="$configdir/all/retroarch/cores"
     # remove if not a git repository and do a fresh checkout
     [[ ! -d "$dir/.git" ]] && rm -fr "$dir"
+    # remove our locally generated `.info` files, just in case upstream adds them
+    [[ -d "$dir/.git" ]] && git -C "$dir" clean -q -f "*.info"
     gitPullOrClone "$configdir/all/retroarch/cores" https://github.com/libretro/libretro-core-info.git
-    # Add the info files for cores/configurations not available upstream
-    cp -f "$md_data/"*.info "$configdir/all/retroarch/cores"
+    # add our info files for cores not included in the upstream repo
+    for info_file in "$md_data"/*.info; do
+        [[ ! -f "$configdir/all/retroarch/cores/$(basename $info_file)" ]] && \
+            cp "$info_file" "$configdir/all/retroarch/cores/"
+    done
```

Comment:
What's the reason for this change over the previous copy command ?

Reply:
To not overwrite any files that may be added to the repo at a later date ? Maybe cp -u would be enough to handle that (Since the date would be newer). But if this change is needed in case I missed something it would be best to use a bash built-in over basename (as we do elsewhere).

Reply:
Also it's missing a `local`

Reply:
> To not overwrite any files that may be added to the repo at a later date ?...

Yes, that was the intention. `cp --update` works better, thanks.